docs/example_usecases.md

```diff
@@ -0,0 +1,15 @@
+<!--
+SPDX-FileCopyrightText: 2023 Springtime authors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+Here, we provide several notebooks to show how to use `springtime` package in
+different use cases. Click on each title to download the notebook.
+
+## Data exploration
+
+- [NPN](notebooks/mk_data_exploration_npn.ipynb)
+- [PEP725](notebooks/mk_data_exploration_pep725.ipynb)
+- [PPO](notebooks/mk_data_exploration_ppo.ipynb)
+- [Daymet](mk_data_exploration_daymet.ipynb)
```

Comment:
```suggestion
- [Daymet](notebooks/mk_data_exploration_daymet.ipynb)
```

Reply:
This page is removed, [as suggested by Peter](https://github.com/phenology/springtime/pull/105#discussion_r1154054637).